content/workflows/csp/aws/service/rds/resources/db/policy-context/rds-tls-disabled.yaml

```
@@ -0,0 +1,27 @@
+schema: 1
+type: policy-context
+
+id: rds-tls-disabled
```

```suggestion
id: rds-tls-is-disabled
```

Rename file as well.